src/main/java/goodgenerator/loader/NaquadahReworkRecipeLoader.java

```diff
@@ -58,25 +63,26 @@ public static void RecipeLoad() {
                             Materials.Gallium.getDust(64)
                         },
                         new int[] {2500, 2500, 2500, 2500},
-                        20 * 20,
-                        (int) GT_Values.VP[9],
+                        10 * 20,
+                        (int) GT_Values.VP[10],
                         2);
                 // Enriched Naquadah (UIV)
                 CORE.RA.addQuantumTransformerRecipe(
                         new ItemStack[] {
                             enrichedNaquadahEarth.get(OrePrefixes.dust, 32),
                             Materials.Zinc.getDust(64),
+                            Materials.Carbon.getDust(1),
                             GT_Utility.copyAmount(0, GenericChem.mAdvancedNaquadahCatalyst)
                         },
-                        new FluidStack[] {Materials.SulfuricAcid.getFluid(16000), P507.getFluidOrGas(10)},
+                        new FluidStack[] {Materials.SulfuricAcid.getFluid(16000), Materials.Oxygen.getGas(100L)},
```

Comment:
why is the oxygen cost only 100?

Reply:
Only P507 has Oxygen, and it was used in the recipe in even smaller amounts.